Bigtable/src/Instance.php

```diff
@@ -240,17 +241,16 @@ public function reload(array $options = [])
      *     Configuration options
      *
      *     @type string $displayName **Defaults to** the value of $instanceId.
-     *     @type int $instanceType  Possible values include `Instance_Type::PRODUCTION` and `Instance_Type::DEVELOPMENT`.
-     *           **Defaults to** `Instance_Type::TYPE_UNSPECIFIED`.
-     *     @type array $labels For more information, see
+     *     @type int $instanceType Possible values include @var Instance::INSTANCE_TYPE_PRODUCTION  and @var Instance::INSTANCE_TYPE_DEVELOPMENT.
```

Comment:
WDYT about the following (and similar elsewhere we reference constants). I'm not sure the `@var` tag will render correctly on the docs site.

```
Possible values are represented by the following constants: `Google\Cloud\Bigtable\Instance::INSTANCE_TYPE_PRODUCTION`, `Google\Cloud\Bigtable\Instance::INSTANCE_TYPE_DEVELOPMENT`, and `Google\Cloud\Bigtable\Instance::INSTANCE_TYPE_UNSPECIFIED`. **Defaults to** using `Google\Cloud\Bigtable\Instance::INSTANCE_TYPE_UNSPECIFIED`.
```